multibase.csv

```diff
@@ -2,11 +2,15 @@ encoding,     code, name
 base1,        1,    unary tends to be 11111
 base2,        0,    binary has 1 and 0
 base8,        7,    highest char in octal
-base10,        9,    highest char in decimal
+base10,       9,    highest char in decimal
 base16,       Ff,   highest char in hex
-base32,       Uu,   rfc4648 - highest letter
-base32hex,    Vv,   rfc4648 - highest char
-base58flickr, Z,    highest char
-base58btc,    z,    highest char
-base64,       y,    rfc4648 highest char
-base64url,    Y,    rfc4648 highest char
+base32hex,    Vv,   rfc4648 no padding - highest char
+base32,       Bb,   rfc4648 no padding
+base32,       Cc,   rfc4648 with padding
```

Comment:
I think we want to label these differently, to make implementations nicer (unique base name). Maybe `base32pad` ?